app/styles/app.css

```diff
@@ -96,6 +96,11 @@ hr {
   letter-spacing: 0.1em;
 }
 
+.cf-card-field {
+  text-overflow: ellipsis;
+  max-width: 20rem;
+}
+
```

Comment:
this fixes a bug where, if you have a really long string (like a long url) the edit button disappears